tools/fp20compiler/Makefile

```diff
@@ -1,7 +1,15 @@
 MAIN = fp20compiler
 CXXFLAGS= -DYY_NO_UNPUT -Wall -g
 
-INCLUDES = $(wildcard *.h) \
+INCLUDES = nvparse_errors.h \

```

Comment:
nit: Weird indentation. Start this with an empty first line which just has `\`. Otherwise it's easy to miss "nvparse_errors.h"

(Other instances, too)